Hangman/Hangman/main.swift

```diff
@@ -7,6 +7,191 @@
+    }
+    return currentAlphabet //string, contains, removeoccurrences
+}
+
+func checkWordComplete(checkWord : String, soFarWord : String) -> Bool    {
+    var count = 0
+    var complete = false
+    for char in checkWord {
+        if soFarWord.contains(char)   {
+            count += 1
+        }
+    }
+    if count == checkWord.count {
+        complete = true
+    }
+    else {complete = false}
+    return complete
+}
+
+func updateBodyParts(guess : Bool) -> Int  {
+    if guess == false   {
+        updatedBodyParts -= 1
+    }
+    return updatedBodyParts
+}
+
+
+print(greeting + "\n")
+print(hangmanPicking + "\n")
+print(explanation + "\n")
+
+//print starting board
+for _ in cpuWord {
+    hiddenWord += (blankSpace) + " "
+}
+
+print(hiddenWord + "\n")
+print("\n" + alphabet)
+
+
+//start loop after intro print for single game
+//loop whole thing for play again
+
+repeat  {
+let userGuess = String(readLine() ?? "")
+guess = guessOutput(enteredLetter: userGuess)
+
+//var remainingLetters = remainingAlphabet(enteredLetter: userGuess) //move adjust
+
+    let alreadyEntered = checkAlreadyEntered(enteredLettersArray: enteredLetters, enteredLetter: userGuess) //check print
+
+bodyPartsLeft = updateBodyParts(guess: guess)
+    
+if alreadyEntered == true   {
+    print("Already entered letter")
+    print("\n" + currentWord + "\n")
+    print("You have \(bodyPartsLeft) body parts left")
+    print("Used Letters: \(enteredLetters)")
+    alphabet = remainingAlphabet(enteredLetter: userGuess, alph: alphabet)
+    print("Remaining Letters: \(alphabet)\n")
+}
+
+else if guess == guessedRight    {
+    currentWord = updateRight(enteredLetter: userGuess)
+    enteredLetters = enteredLetterBucket(enteredLetter: userGuess)
+    print("\n" + currentWord + "\n")
+    print("You have \(bodyPartsLeft) body parts left")
+    print("Used Letters: \(enteredLetters)")
+    alphabet = remainingAlphabet(enteredLetter: userGuess, alph: alphabet)
+    print("Remaining Letters: \(alphabet)\n")
+    if checkWordComplete(checkWord: cpuWord, soFarWord: currentWord) == true    {
+        print("You Win!")
+        print(cpuWord)
+        gameOver = true
+    }
+    //store used alphabet letters
+    //print remaining alphabet
+    //check if word completed or out of chances to end game
+}
+    
+else    {
+    currentWord = updateWrong()
+    enteredLetters = enteredLetterBucket(enteredLetter: userGuess)
+    print("\n" + currentWord + "\n")
+    print("You have \(bodyPartsLeft) body parts left")
+    print("Used Letters: \(enteredLetters)")
+    alphabet = remainingAlphabet(enteredLetter: userGuess, alph: alphabet)
+    print("Remaining Letters: \(alphabet)\n")
+    if bodyPartsLeft == 0   {
+        print("You Lose")
+        print(cpuWord)
+        gameOver = true
+    }
+}
+}  while gameOver == false
+
+print("Play Again? (y for Yes, n for No)")
+    playResponse = readLine() ?? ""
+} while playResponse == "y"
```

Comment:
There was a bug where it would only allow me to replay the game twice, it also would dip into negative body parts, so it would say "you have -9 body parts left" and it would not let me loose the game and/or display the correct hidden word.